src/collective/easyform/profiles/default/actions.xml

```diff
@@ -3,6 +3,29 @@
         meta_type="Plone Actions Tool"
         name="portal_actions"
 >
+  <object meta_type="CMF Action Category"
+          name="object"
+  >
+    <object meta_type="CMF Action"
+            name="easyform-view"
```

Comment:
This will add a 'puzzle piece' icon to the toolbar. Can you please add these three lines to easyform.css?

.icon-easyform-view.toolbar-menu-icon::before {
    content: '\e807';
}

I don't think an upgrade step is necessary, but am not sure. When I locally start a test site with this change the easyform.css is bundled into the default.css automatically. But maybe we should upgrade the timestamp in the IBundleRegistry GS for this resource ....

Reply:
You added the css.
When running in `bin/instance console`, I do not see a result of your changes, so an upgrade step is needed. But that is trickier than it should be.
I will merge now, and worry about an upgrade step afterwards.